hazelcast-integration/operator-framework/operator.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hazelcast-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: hazelcast-operator
+  template:
+    metadata:
+      labels:
+        name: hazelcast-operator
+    spec:
+      serviceAccountName: hazelcast-operator
+      containers:
+        - name: hazelcast-operator
+          image: hazelcast/hazelcast-operator:0.0.1
+          imagePullPolicy: Always
+          env:
+            - name: WATCH_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: POD_NAME
```

Comment:
where is this pod_name used?

Reply:
It's auto-generated. We don't use it anywhere.